boards/arm/tmo_dev_edge/doc/index.rst

```diff
@@ -0,0 +1,225 @@
+| PB11  |   6 | UART_TX     | UART_TX console LEU0_TX             |
++-------+-----+-------------+-------------------------------------+
+| PB12  |   6 | UART_RX     | UART_RX console LEU0_RX             |
++-------+-----+-------------+-------------------------------------+
+| PA1   |   1 | I2C_SDA     | ENV_I2C_SDA I2C0_SDA                |
++-------+-----+-------------+-------------------------------------+
+| PA0   |  31 | I2C_SCL     | ENV_I2C_SCL I2C0_SCL                |
++-------+-----+-------------+-------------------------------------+
+| PJ15  |  12 | I2C_SDA     | ENV_I2C_SDA I2C1_SDA                |
++-------+-----+-------------+-------------------------------------+
+| PJ14  |  10 | I2C_SCL     | ENV_I2C_SCL I2C1_SCL                |
++-------+-----+-------------+-------------------------------------+
+
+
+System Clock
+============
+
+The EFM32PG SoC is configured to use the 38.4 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFM32PG SoC has four USARTs and one Low Energy UART (LEUART).
+
+Programming and Debugging
+*************************
+
+.. note::
+   Before using the kit the first time, you should update the J-Link firmware
+   from `J-Link-Downloads`_
+
+Flashing
+========
+
+A `J-Link`_ serial and debug adaptor is required. The adaptor provides:
+
+Flashing an application to tmo_dev_edge
+------------------------------------------
+
+The sample application :ref:`hello_world` is used for this example.
+Build the Zephyr kernel and application:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: tmo_dev_edge
+   :goals: build
+
+Connect the tmo_dev_edge to your host computer using the USB port and you
+should see 4 USB serial connections
+
+Use a USB-A to Mini-USB cable to connect to the UARTs on the board.
+
+Open a serial terminal (minicom, putty, etc.) with the following settings:
+
+- Port: 2nd of 4
+- Speed: 9600
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Reset the board and you'll see the following message on the corresponding serial port
+terminal session:
+
+.. code-block:: console
+
+   Hello World! arm
+
+Fixtures implemented on the tmo_dev_edge board
+-------------------------------------------------
+
+The picture of this board above shows a Saleae probe connected between PK0 and PK1
+test points on the 820-0013-00_silabs_cpu_board_2021-03-22 feather board. These are
+unused GPIO pins that form a fixture for the following tests:
+
+/tests/drivers/gpio/gpio_basic_api - gpio_loopback
+tests/drivers/regulator/fixed - regulator_loopback
+
+The Twister tests can be run from the boards folder as follows:
+
+.. code-block:: console
+
+   ../../../../zephyr/scripts/twister --device-testing \
+                                      --hardware-map tmo_dev_edge-map.yml \
+                                      --load-tests tmo_dev_edge-testplan.json
+
+2 of 8 regulator tests are still failing (due to 9600 baud issues), but the gpio tests work fine.
```

Comment:
Have you tried `CONFIG_ZTEST_TEST_DELAY_MS` different from zero?

Reply:
I have not tried that yet, but thanks for the pointer. I have determined that it is not strictly a 9600 baud problem, since the efm32pg_stk3402a is running at 115200 baud and fails in similar places, but adding time between tests might be a common solution. Thanks!